worlds/rabi_ribi/__init__.py

```diff
@@ -38,42 +38,17 @@ class RabiRibiWorld(World):
     topology_present: bool = False
     web: WebWorld = RabiRibiWeb()
 
-    base_id: int = 8350438193300
+    base_id: int = rabi_ribi_base_id
 
-    item_name_groups: Dict[str, Set[str]] = {}
+    item_name_groups = item_groups
     location_name_groups: Dict[str, Set[str]] = {}
 
-    item_name_to_id: Dict[str, int] = {
-        name: id_num for
-        id_num, name in enumerate(item_set, base_id)
-    }
-    item_name_to_id["Nothing"] = base_id + len(item_name_to_id)
+    item_name_to_id = {name: data.code for name, data in item_table.items()}
     location_name_to_id: Dict[str, int] = {
         name: id_num for
         id_num, name in enumerate(get_all_possible_locations(), base_id)
```

Comment:
You don't need to do so on this PR, but heads up the locations are going to have the same problem if anyone ever makes the locations variable.

Reply:
Makes sense; I created some new tables in `locations.py` for all the locations, including the post game and gift items for the future. I also added a names module for item and location names.